onnx/reference/ops/op_reduce_l1.py

```diff
@@ -3,11 +3,32 @@
 
 import numpy as np
 
+from onnx.defs import onnx_opset_version
+
 from ._op import OpRunReduceNumpy
 
 
-class ReduceL1(OpRunReduceNumpy):
+class ReduceL1_1_11_13(OpRunReduceNumpy):
```

Comment:
I should highlights the current convention, it shoud be <op name>_<op_version>. op_version is the opset it is defined. It is then valid from op_version to until there is a nore recent one. So in your case, it should be `ReduceL1_1`.

Reply:
Agree about factorizing it so we can define `ReduceL1_1`, `ReduceL1_11` etc. using a couple of lines.